src/components/features/occasion-filter/occasion-filter.tsx

```
@@ -0,0 +1,68 @@
+import { useTranslations } from "next-intl";
+import { useFormContext } from "react-hook-form";
+import BtnReset from "@components/shared/btn-reset";
+import OccasionCard from "./components/occasion-card";
+import { getAllOccasions } from "@lib/apis/occasions-filter.api";
+
+export default function OccasionFilter() {
+  const t = useTranslations("Occasions");
+  const { watch, setValue } = useFormContext();
+  const selected = watch("occasionIds") || [];
+
+  const [occasions, setOccasions] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)